app/src/main/res/layout/fragment_events.xml

```diff
@@ -41,26 +41,18 @@
 
   <LinearLayout
     android:id="@+id/empty"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_centerInParent="true"
```

Comment:
centerInParent is I think needed because we are using this in both RelativeLayout and FrameLayout time to time.